File: problems/900/318A_Even_Odds_CF.py
 # Codeforces 318A - Even Odds (900)


def main():
    n_str = input().split()
    if len(n_str) != 2:
        return
    n, k = map(int, n_str)

    odds = (n + 1) // 2
    if k <= odds:
        ans = 2 * k - 1
    else:
        ans = 2 * (k - odds)

    print(ans)


if __name__ == "__main__":
    main()

File: problems/900/318A_Even_Odds_CF.js
// Codeforces 318A - Even Odds (900)

'use strict';

const fs = require('fs');

function main() {
    const data = fs.readFileSync(0, 'utf8').trim();
    if (!data) return;

    const parts = data.split(/\s+/);
    if (parts.length < 2) return;

    const n = BigInt(parts[0]);
    const k = BigInt(parts[1]);

    const odds = (n + 1n) / 2n;
    let ans;
    if (k <= odds) {
        ans = 2n * k - 1n;
    } else {
        ans = 2n * (k - odds);
    }

    process.stdout.write(ans.toString());
}

main();

